setup.py

```diff
@@ -111,6 +111,11 @@
         "numpy",
         "pre-commit"
     ],
+    "test-http-v2": [
+        "azurefunctions-extensions-http-fastapi",
+        "ujson",
+        "orjson"
```

Comment:
Why do we add these json libs?

Reply:
they are required to support fastapi UJSONResponse and ORJSONResponse types. fastapi wont install them itself. if customer wants to use then they install these packages separately. here is for testing
![image](https://github.com/Azure/azure-functions-python-worker/assets/12449837/2992b5d6-f091-40e6-a31d-d0a640318523)